src/main/scala/org/virtuslab/internal/load/construct/YamlDecoder.scala

```diff
@@ -9,29 +8,110 @@ import scala.compiletime._
 import scala.compiletime.summonFrom
 import scala.deriving._
 import scala.util.Try
+import scala.language.implicitConversions
```

Comment:
I would prefer explicit conversion rather than an implicit one because it's.. more explicit and less magical

Reply:
Good point, it should be removed immediately :)

Reply:
Sure, I removed implicit conversion